src/modules/fancyzones/lib/Zone.cpp

```diff
@@ -96,7 +99,9 @@ void Zone::SizeWindowToZone(HWND window, HWND zoneWindow) noexcept
         const auto taskbar_left_size = std::abs(mi.rcMonitor.left - mi.rcWork.left);
         const auto taskbar_top_size = std::abs(mi.rcMonitor.top - mi.rcWork.top);
         OffsetRect(&newWindowRect, -taskbar_left_size, -taskbar_top_size);
-        if (accountForUnawareness)
+
+        int monitorCount = MonitorInfo::GetMonitors(true).size();
```

Comment:
We don't need to call `GetMonitors` if `accountForUnawareness` is false (and it is false most of the times).
Move it inside the `if` statement.

Reply:
I've change the structure of if so GetMonitors won't be called when accountForUnawareness is false